includes/polling/os/ciscosb.inc.php

```diff
@@ -13,3 +13,7 @@
 $hardware = str_replace(' ', '', snmp_get($device, 'CISCOSB-Physicaldescription-MIB::rlPhdUnitGenParamModelName.1', '-Ovq'));
 $serial   = snmp_get($device, 'CISCOSB-Physicaldescription-MIB::rlPhdUnitGenParamSerialNum.1', '-Ovq');
 $features = snmp_get($device, 'CISCOSB-Physicaldescription-MIB::rlPhdUnitGenParamServiceTag.1', '-Ovq');
+
+if ($poll_device['sysObjectID'] == '.1.3.6.1.4.1.9.6.1.89.26.1') {
```

Comment:
Might seem silly on this one but it's worth moving that above the current hardware and doing an else so we avoid an snmp_get request for no reason if it's not needed / used.